docs/languages/csharp/delegates-events.md

```diff
@@ -2,4 +2,42 @@
 
 > **Note**
 > 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+> This topic will be covered under the following articles:
+
+<style type="text/css">
```

Comment:
I haven't used this with docfx, but is it appropriate to place this here?  It shows as text in the markdown on plain 'ole Github.